### ISSUE TITLE
sitemap.xml can't be fetched by searchbots

### ISSUE THREAD

**Issue by Joduai** [bludit/bludit#1022]
After I raised my page, I tried to add sitemap.xml in google's search console (earlier webmasters tool). After it couldn't fetch the sitemap I assumed that new version of google's service doesn't always work as supposed. Two months later I attempted again to fetch the sitemap after I've noticed that almost none of the pages were indexed in search engines.

I checked if any type of bot could access the /sitemap.xml
While sitemap.xml can be accessed without any problem in browsers I can't find out what blocks the bots from accessing the sitemap.

Cold somebody check using below example tools if sitemaps on your pages can be fetched and processed?
https://www.xml-sitemaps.com/se-bot-simulator.html
http://sitemapinspector.com/

I'm not fond of redirecting all traffic to be processed by index.php file in htaccess, as it causes some problems with webapps stored in nested directories under domain directory.
But after disabling the proper rewrite rule, a physically existing sitemap.xml file can be accessed but still won't be fetched, as the URLs inside are non existant.

**Comment by dignajar** [comment]
Hi,
I'm checking this sitemap.xml with the site you mention and I have the same errors.
- https://www.cloudflare.com/sitemap.xml
- https://wordpress.com/sitemap.xml
- https://demo.bludit.com/sitemap.xml

The last links is running Bludit.

**Comment by Joduai** [comment]
I suppose that those tools aren't working at all, as only one of website's I checked lately was processed.

I wouldn't be surprised with bugs in google's search console. But on another website on same hosting account when I generate the sitemap periodically saving it as a file, googlebot/search console has no problems with fetching it, and indexing all links inside it contains.

Just tried with adding an exception rule to let an existing sitemap.xml be processed, so that all other traffic would still be processed by index.php. No use, sitemap cannot be fetched. Even one I generated with 2 online generators. Apache log doesn't show any access errors.

This starts to look like a Czech movie :)
